protocols/kad/src/record/store.rs

```diff
@@ -64,36 +66,33 @@ pub enum Error {
 ///      content. Just like a regular record, a provider record is distributed
 ///      to the closest nodes to the key.
 ///
-pub trait RecordStore<'a> {
```

Comment:
Shouldn't `'a` have been on the individual methods instead of the entire `trait` definition?